src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -365,6 +386,16 @@ const PermissionIdentities: FC = () => {
           }}
         />
       )}
+
+      {isSsoModal && (
+        <Modal
+          close={closeSsoModal}
+          title="SSO setup"
+          className="choose-sso-provider-modal"
+        >
+          <SsoProviderSelector />
+        </Modal>
+      )}
```

Comment:
We should best contain this logic within the `SsoNotification` component, so it triggers the modal, and also renders it within a portal. So we don't leak details like state handling for the modal and import of components into this component.